test/framework/integration_framework/iroha_instance.hpp

```diff
@@ -19,28 +19,43 @@
 #define IROHA_IROHA_INSTANCE_HPP
 
 #include <cstdlib>
-#include "integration/pipeline/tx_pipeline_integration_test_fixture.hpp"
+
+#include "integration/pipeline/test_irohad.hpp"
 #include "main/raw_block_insertion.hpp"
 
 namespace integration_framework {
 
   using namespace std::chrono_literals;
 
+  /**
+   * Class provides interfaces for running Iroha instance.
+   */
   class IrohaInstance {
    public:
-    void makeGenesis(const iroha::model::Block &block) {
+    /**
+     * Save the block as genesis block.
+     * @param block - block to be saved as genesis block
+     */
+    void insertGenesis(const iroha::model::Block &block) {
```

Comment:
Since this method drops ledger, then inserts block, maybe it would be wise to name it something like `dropThenInsertGenesis`?

Reply:
`replaceGenesis` is simplier i guess